.pre-commit-config.yaml

```diff
@@ -52,7 +52,6 @@ repos:
     rev: 24.4.2
     hooks:
     -   id: black
-        args: ["--check", "--diff", "--color"]
```

Comment:
Just curious, was this change intentional?

Reply:
Yes.
If black is supplied with any of these arguments it stops reformat files.
Without these arguments black would reformat files and return error if any file was reformatted.
So it works on CI and can be run locally to automatically reformat files.